utilities/include/alps/numeric/tensors/data_storage.hpp

```diff
@@ -20,45 +21,94 @@ namespace alps {
        * @brief TensorBase class implements basic multi-dimensional operations.
        * All arithmetic operations are performed by Eigen library
        */
```

Comment:
Now that you have two template parameters, it might be good to briefly explain what they do. Something to the effect of

```C++
  /**
   * @tparam T  the scalar type
   * @tparam Cont  abstraction of the data storage (default vector)
   */
```